h/zos.h

```diff
@@ -1157,7 +1196,7 @@ int setTaskAcee(ACEE *acee);
 
 int safVerify(int options, char *userid, char *password,
               ACEE **aceeHandle,
-              int *racfStatus, int *racfReason);
+              int *racfStatus, int *racfReason, Idta *idta);
```

Comment:
`safVerify` is a public function, we can't change its arguments because it'll break the users.

Reply:
Fixed